app/(tabs)/movie/[id]/seats.tsx

```diff
@@ -54,14 +112,47 @@ export default function ChooseSeatScreen() {
         >
           <Ionicons name='arrow-back' size={20} color={iconColor} />
         </Pressable>
-        <ThemedText
-          style={[styles.movieTitle, { color: textPrimaryColor }]}
-          numberOfLines={1}
-        >
-          {bookingData?.movieTitle || 'Movie'}
-        </ThemedText>
-        <View style={styles.placeholder} />
+
+        {/* Movie info section */}
+        <View style={styles.movieInfoContainer}>
+          {movieImage && (
+            <Image
+              source={{ uri: movieImage }}
+              style={styles.movieThumbnail}
+              resizeMode='cover'
+            />
+          )}
+          <View style={styles.movieInfoText}>
+            <ThemedText
+              style={[styles.movieTitle, { color: textPrimaryColor }]}
+              numberOfLines={2}
+            >
+              {bookingData?.movieTitle || movie?.title || 'Movie'}
+            </ThemedText>
+            {showtimeData && (
+              <ThemedText
+                style={[styles.showtimeInfo, { color: textSecondaryColor }]}
+              >
+                {formatShowtime()} in {formatDate()}
+              </ThemedText>
+            )}
+          </View>
+        </View>
       </View>
+
+      {/* Seat Layout */}
+      <ScrollView
+        style={styles.scrollView}
+        contentContainerStyle={styles.scrollContent}
+        showsVerticalScrollIndicator={false}
+      >
```

Comment:
The `SeatLayout` already has a `ScrollView` can you check if this one is necessary?